composer.json

```diff
@@ -22,7 +22,6 @@
         "source": "https://github.com/cakephp/twig-view"
     },
     "require": {
-        "php": ">=7.2",
```

Comment:
Personally I always liked the verbosity of this, since this needs to match the min/max of CI and people can more easily perceive the min range.
This can also easily change (higher than core setting) etc.

Reply:
This causes tools like psalm to run in compatibility mode for older PHP version even when using PHP 7.4. It shows "Target PHP version: 7.2 (inferred from composer.json)".

Reply:
I see

Reply:
https://github.com/vimeo/psalm/issues/8130#issuecomment-1161421242